### DISCUSSION TITLE
Layouts DragDrop

### DISCUSSION THREAD

**Discussion by ham187** [dotnetcore/BootstrapBlazor#3759] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

If you drag the bottom item, down and it doesnt drop anywhere all the items are set to class= "bb-dd-draggable bb-dd-inprogess" and the bottom item is set to: class="bb-dd-draggable no-pointer-events bb-dd-inprogess" 

### Expected Behavior

it should set back to:  class="bb-dd-draggable "
If you have for example a dropdown or a form control you can't use it anymore.

### Steps To Reproduce

You can use the live page at: https://www.blazor.zone/drag-drop

1. in Basic usage grab number 10 and drag it down to show code row, and release then look at the class

### .NET Version

8

**Comment by bb-auto** [comment]
@ham187 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.
